ipa.go

```diff
@@ -31,10 +31,13 @@ import (
 )
 
 type (
-	Fr    = fr.Element
-	Point = banderwagon.Element
+	Fr                    = fr.Element
+	Point                 = banderwagon.Element
+	SerializedAffinePoint = banderwagon.SerializedAffinePoint
 )
 
+const SerializedAffinePointSize = banderwagon.SerializedAffinePointSize
```

Comment:
New type and constant from go-ipa that are pulled here to be more explicit and type safe.